cdc/tests/utils/rawkv_data/gen_data.go

```diff
@@ -0,0 +1,207 @@
+		keys = append(keys, key)
+		values0 = append(values0, value0)
+		values1 = append(values1, value1)
+	}
+	return keys, values0, values1
+}
+
+func NewPutCommand() *cobra.Command {
+	return &cobra.Command{
+		Use:          "put",
+		Short:        "Put rawkv data into tikv",
+		SilenceUsage: true,
+		Args:         cobra.NoArgs,
+		RunE: func(cmd *cobra.Command, _ []string) error {
+			return runPointPut(cmd)
+		},
+	}
+}
+
+func NewDeleteCommand() *cobra.Command {
+	return &cobra.Command{
+		Use:          "delete",
+		Short:        "Delete rawkv data from tikv",
+		SilenceUsage: true,
+		Args:         cobra.NoArgs,
+		RunE: func(cmd *cobra.Command, _ []string) error {
+			return runPointDelete(cmd)
+		},
+	}
+}
+
+func runPointDelete(cmd *cobra.Command) error {
+	cfg := &Config{}
+	err := cfg.ParseFromFlags(cmd.Flags())
+	if err != nil {
+		return err
+	}
+
+	ctx := context.Background()
+	cli, err := rawkv.NewClientWithOpts(ctx, []string{cfg.SrcPD}, rawkv.WithAPIVersion(kvrpcpb.APIVersion_V2))
+	if err != nil {
+		return err
+	}
+	defer cli.Close()
+
+	eg := new(errgroup.Group)
+	for i := cfg.StartIndex; i < cfg.Count+cfg.StartIndex; i++ {
+		i1 := i
+		eg.Go(func() error {
+			key, _, _ := generateTestData(i1)
+			err = cli.Delete(ctx, key)
+			if err != nil {
+				return err
+			}
+			return nil
+		})
+	}
+	err = eg.Wait()
+	if err != nil {
+		log.Info("failed to delete data", zap.Error(err), zap.Int("delete count", cfg.Count))
+	} else {
+		log.Info("delete data successfully", zap.Error(err), zap.Int("delete count", cfg.Count))
+	}
+	return err
+}
+
+func runPointPut(cmd *cobra.Command) error {
+	cfg := &Config{}
+	err := cfg.ParseFromFlags(cmd.Flags())
+	if err != nil {
+		return err
+	}
+	ctx := context.Background()
+
+	cli, err := rawkv.NewClientWithOpts(ctx, []string{cfg.SrcPD}, rawkv.WithAPIVersion(kvrpcpb.APIVersion_V2))
+	if err != nil {
+		return err
+	}
+	defer cli.Close()
+
+	startIdx := cfg.StartIndex
+
+	rand.Seed(uint64(time.Now().UnixNano()))
+	count1 := rand.Intn(cfg.Count)
+	count2 := rand.Intn(cfg.Count - count1)
+
+	eg := new(errgroup.Group)
+	eg.Go(func() error {
+		startIdx1 := startIdx
+		endIdx := startIdx + count1
+		for i := startIdx1; i < endIdx; i++ {
+			key, value0, value1 := generateTestData(i)
+			err = cli.Put(ctx, key, value0)
+			if err != nil {
+				return err
+			}
+			err = cli.Put(ctx, key, value1)
+			if err != nil {
+				return err
+			}
+		}
+		return nil
+	})
+
+	eg.Go(func() error {
+		startIdx1 := startIdx + count1
+		endIdx := startIdx + count1 + count2
+		kvCntPerBatch := 512
+		for startIdx1 < endIdx {
+			batchCnt := min(kvCntPerBatch, endIdx-startIdx)
+			keys, values0, values1 := batchGenerateData(startIdx1, batchCnt)
+			err := cli.BatchPut(ctx, keys, values0)
+			if err != nil {
+				return err
+			}
+			err = cli.BatchPut(ctx, keys, values1)
+			if err != nil {
+				return err
+			}
+			startIdx1 += kvCntPerBatch
+		}
+		return nil
+	})
+
+	cli.SetAtomicForCAS(true)
```

Comment:
`SetAtomicForCAS` will also affect the two routines before. Suggest to use a new rawkv client.